docs/user_guides/transformer_workflow.rst

```diff
@@ -10,47 +10,49 @@ Pyreal introduces specific terms and naming schemes to refer to different featur
++-------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``x_algorithm``, ``explanation_algorithm``| Data or an explanation using the feature space expected by the explanation algorithm (explanation-ready feature space)                                  |
++-------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``x_model``, ``explanation_model``        | Data or an explanation using the feature space the model expects                                                                                        |
++-------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``x_interpret``,                          |                                                                                                                                                         |
+| ``explanation_interpret``                 | Data or an explanation using the most human-readable feature space available                                                                            |
++-------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``model transformers``                    | Transformers that transform data between the original feature space to the model-ready feature space. Transformers with ``model=True`` flag.            |
++-------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``algorithm transformers``                | Transformers that transform data and explanations between the original feature space to the explanation-algorithm-ready feature spaces.                 |
+|                                           | Transformers with ``algorithm=True`` flag.                                                                                                              |
++-------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``i_transformers``                        | Transformers that transform data and explanations between the original feature space to the interpretable feature space.                                |
```

Comment:
should this be renamed to `interpret transforms` for consistency?

Reply:
Yes, good catch, missed this